e2e-app/src/app/datepicker/autoclose/datepicker-autoclose.e2e-spec.ts

```diff
@@ -1,6 +1,6 @@
 import {Key} from 'protractor';
 import {DatepickerAutoClosePage} from './datepicker-autoclose.po';
-import {openUrl, sendKey} from '../tools';
+import {openUrl, sendKey} from '../../tools.po';
```

Comment:
didn't get why you renamed `tools.ts` → `tools.po.ts` as its just helper functions and not a page object. ex `openUrl`?